src/Microsoft.PowerShell.ConsoleGuiTools/Microsoft.PowerShell.ConsoleGuiTools.psd1

```diff
@@ -9,7 +9,8 @@
 RootModule = 'Microsoft.PowerShell.ConsoleGuiTools.dll'
 
 # Version number of this module.
-ModuleVersion = '0.7.4'
+# NOTE: This will get updated by build.ps1; the build number will be incremented for each build.
+ModuleVersion = '0.7.5.30'
```

Comment:
Why do we need to add another field to the version?

Reply:
Because semver. 

I need this for local development (otherwise I can't tell for sure what got loaded given PS makes it really hard to unload modules and dependent libs).

Reply:
Got it, makes sense!